experimenter/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -583,29 +577,19 @@ describe("FormAudience", () => {
     fireEvent.change(screen.getByTestId("targetingConfigSlug"), {
       target: { value: MOCK_CONFIG!.targetingConfigs![0]!.value },
     });
-    fireEvent.click(screen.getByTestId("isFirstRun"), {
-      target: { checked: false },
-    });
-    expect(screen.getByTestId("isFirstRun")).toHaveProperty("checked", true);
-    expect(screen.getByTestId("isFirstRun")).not.toBeDisabled();
-    await expect(
-      screen.queryByTestId("is-first-run-required-warning"),
-    ).not.toBeInTheDocument();
```

Comment:
same as here, removing all the expects, that means this test is testing nothing, so would suggest modify it, we still want to test the same behaviour that is written in test description